.github/workflows/build-client-server.yml

```
@@ -264,10 +512,128 @@ jobs:
+        if: needs.ci-test-full.result != 'success'
+        shell: bash
+        run: |
+          new_failed_spec_env="<ol>$(comm -1 -3 <(sort ~/knownfailures) <(sort -u ~/combined_failed_spec_ci) | sed 's/|cypress|cypress/\n/g' | sed 's/^/<li>/')</ol>"
+          echo "$new_failed_spec_env"
+          echo "new_failed_spec_env<<EOF" >> $GITHUB_ENV
+          echo "$new_failed_spec_env" >> $GITHUB_ENV
+          echo "EOF" >> $GITHUB_ENV
+
+      - name: Add a comment on the PR with new CI failures
+        if: needs.ci-test-full.result != 'success' && needs.file-check.outputs.pr != '0'
+        uses: peter-evans/create-or-update-comment@v3
```

actionlint (1.7.12)

[error] the runner of "peter-evans/create-or-update-comment@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)